library/components/TDropdown.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div
 		class="relative font-sans cursor-pointer w-max dropdown-filter"
-		@mouseover="setActiveState"
-		@mouseleave="setActiveState($event, false)"
+		@mouseover="hasMouseOver = true"
```

Comment:
When dropdown has child dropdown under, clicking on it is considered clicking outside which closes parent dropdown, this fix makes sure that does not happen.

Reply:
Have you considered using event modifiers to overcome this bug? 

```
<div @click.self="doThat">...</div>
```
https://vuejs.org/guide/essentials/event-handling.html#event-modifiers


I only mention it because ProductCL also has to avoid closing the main dropdown when there are filters inside, and they have a recipe here that makes use of event modifiers to stop the event when necessary: https://productcl.netlify.app/?path=/docs/recipes-dropdownfilters--overview

I'm not sure if it solves your problem though.

I am

Reply:
Woow, yup that seems to be working, let me test it a bit then I'll push it, thanks @sareg0 .

Reply:
NICE!

Reply:
Alright seems to be working perfectly, thanks again, not sure why I never knew about it lol

Reply:
Sadly we still need the height fix for a different issue by product cl, what happens is that if dropdown is invisible the t-tags filter fails to assign the exact selected items and shows +4 for all selected items instead.